apps/frontend/src/app/3_organisms/RuneBridgeDialog/RuneBridgeDialog.tsx

```diff
@@ -39,17 +43,37 @@ export const RuneBridgeDialog: React.FC<RuneBridgeDialogProps> = ({
 }) => {
   const [index, setIndex] = useState(step);
   const { account } = useAccount();
-
+  const { supported: isChainSupported, chainName } = useChainDetails();
   const { isMobile } = useIsMobile();
+
   useEffect(() => {
     setIndex(step);
   }, [step]);
 
   const items = useMemo(() => {
+    if (!isChainSupported) {
+      return [
+        {
+          label: t(translation.tabs.unSupportedLabel, { chainName }),
```

Comment:
```suggestion
          label: '',
```
It doesn't look very good with a label.
<img width="767" alt="image" src="https://github.com/DistributedCollective/sovryn-dapp/assets/14878713/f0395da4-68a1-46d1-92a6-68f795bbc429">